lib/.merlin

```diff
@@ -0,0 +1,19 @@
+B ../_build/default/lib
+FLG -w -40 -w +26
+PKG bigarray
+PKG bytes
+PKG ctypes
+PKG ctypes.foreign
+PKG ctypes.foreign.base
+PKG ctypes.foreign.threaded
+PKG gg
+PKG grenier.binpacking
+PKG integers
+PKG result
+PKG stb_image
+PKG stb_truetype
+PKG tgls.tgles2
+PKG threads
+PKG threads.posix
+PKG tsdl
+PKG unix
```

Comment:
Why not letting JBuilder generate the `.merlin`?

Reply:
I'm never sure if I should let jbuilder generate them. Some editor plugins use the presence of merlin files to determine how to setup the merlin server upon editor / file open. By having one there in place it helps prevent having to restart the editor. However, they should have another mechanism for determining the  correct `cwd` to start merlin in. Searching for the closest merlin file is probably fragile. @let-def can you confirm?

Reply:
> Some editor plugins use the presence of merlin files to determine how to setup the merlin server upon editor / file open. By having one there in place it helps prevent having to restart the editor. 

This is the wrong thing to do for those editors. But in the case where you really want this, then just leave the .merlin file empty in that directory. And let jbuilder generate correct .merlin files in the subdirs.

Reply:
I think @rgrinberg is right, though I don't have a strong opinion there.
Whatever works ;). @jordwalke which editor is it that requires restart?